ci-operator/config/openshift/backplane-cli/openshift-backplane-cli-main.yaml

```
@@ -7,7 +7,7 @@ build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.18
+    tag: golang-1.20
```

I reckon go1.20 tag isn't published yet as well, let's fallback to go1.19.
```suggestion
    tag: golang-1.19
```